main.go

```diff
@@ -81,6 +81,11 @@ func stringFlagOrEnv(p *string, name string, envName string, defaultValue string
 	pflag.StringVar(p, name, defaultValue, usage)
 }
 
+func setEnvLang(lang string, cfg map[string]string) {
+	os.Setenv("AUTO_INSTRUMENTATION_LIMIT_CPU_"+lang, cfg["cpu"])
```

Comment:
what if `cfg["cpu"]` or `cfg["memory"]`  isn't found? We should check to ensure it exists before using it to avoid a panic.